zerver/lib/users.py

```diff
@@ -510,18 +509,18 @@ def format_user_row(
         full_name=row["full_name"],
         timezone=canonicalize_timezone(row["timezone"]),
         is_active=row["is_active"],
-        date_joined=row["date_joined"].isoformat(),
+        # Only send day level precision date_joined data to spectators.
```

Comment:
Should this be "minute level precision"?

Reply:
No; we send day level precision to spectators, and this PR hasn’t changed that.